Source/WebCore/css/calc/CSSCalcTree+Serialization.cpp

```diff
@@ -245,10 +245,10 @@ static Vector<ChildRepresentation, 16> generateSortedChildrenMap(const Children&
 // MARK: Math Function
 // https://drafts.csswg.org/css-values-4/#serialize-a-math-function
 
-static double clampValue(double value, ValueRange range)
+static double clampValue(double value, CSS::Range range)
 {
     value = std::isnan(value) ? 0 : value;
-    return range == ValueRange::NonNegative && value < 0 ? 0 : value;
+    return std::clamp(value, range.min, range.max);
```

Comment:
I normally avoid `std::clamp` because it returns `max` if `max` < `min`, and we typically want `min` in those cases in most call sites. But if that is not an issue here because `CSS::Range` never violates the invariant, then I think it's great to use it.